.cmake-format

```diff
@@ -0,0 +1,277 @@
+# ----------------------------------
+# Options affecting listfile parsing
+# ----------------------------------
+with section("parse"):
+
+  # Specify structure for custom cmake functions
```

Comment:
wow, pls tell me it's generated and you didn't have to put here manually...?

Reply:
no, it is dumped + changed by me where necessary